torchvision/prototype/features/_bounding_box.py

```diff
@@ -115,7 +115,10 @@ def guess_image_size(cls, data: torch.Tensor, *, format: BoundingBoxFormat) -> T
                 data = cls._TO_XYXY_MAP[format](data)
             data = cls._FROM_XYXY_MAP[BoundingBoxFormat.XYWH](data)
         *_, w, h = to_parts(data)
-        return int(h.ceil()), int(w.ceil())
+        if data.dtype.is_floating_point:
+            w.ceil_()
+            h.ceil_()
```

Comment:
Is this change needed? Also, does it change `data` inplace?

Reply:
> Is this change needed?

Yes:

```python
>>> torch.tensor(0, dtype=torch.uint8).ceil()
RuntimeError: "ceil" "_vml_cpu" not implemented for 'Byte'
```

If we decide that images always have a floating point dtype, we can remove it once again.

> Also, does it change `data` inplace?

Yes, good catch. I was under the impression that `.unbind` returned a copy.